test/boost/partitioner_test.cc

```diff
@@ -327,8 +327,6 @@ static
 void
 test_exponential_sharder(const dht::i_partitioner& part, const schema& s, const dht::partition_range& pr) {
 
```

Comment:
Changelog should explain why we can do that.

Reply:
done.